pages/stepprogress/step_progress.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import StepNavigation from "./stepNavigation";
+
+function Step_progess() {
+
+  const labelArray = ['Step 1', 'Step 2', 'Step 3', 'Step 4']
+  const [currentStep, updateCurrentStep] = useState(1);
+
+  function updateStep(step) {
+    updateCurrentStep(step);
+  }
```

Comment:
why do we need the function here?

Reply:
yes we can remove this if we dont intend to have previous step